src/components/TagInfo/TagInfo.tsx

```diff
@@ -0,0 +1,123 @@
+            errorMessage={'Unable to load tag info. Please try again.'}
+        >
+            <TagInfoWrapper>
+                <CollapsibleCard cardTitle={'Main tag info'}>
+                    <InfoRow label="Tag number" value={tagInfo?.tagNo} />
+
+                    <InfoRow label="Description" value={tagInfo?.description} />
+
+                    <InfoRow
+                        label="Register"
+                        value={tagInfo?.registerDescription}
+                        code={tagInfo?.registerCode}
+                    />
+
+                    <InfoRow
+                        label="Tag function"
+                        value={tagInfo?.tagFunctionDescription}
+                        code={tagInfo?.tagFunctionCode}
+                    />
+
+                    <InfoRow
+                        label="System"
+                        value={tagInfo?.systemDescription}
+                        code={tagInfo?.systemCode}
+                    />
+
+                    <InfoRow label="Sequence" value={tagInfo?.sequence} />
+
+                    <InfoRow
+                        label="Discipline"
+                        value={tagInfo?.disciplineDescription}
+                        code={tagInfo?.disciplineCode}
+                    />
+
+                    <InfoRow
+                        label="Area"
+                        value={tagInfo?.areaDescription}
+                        code={tagInfo?.areaCode}
+                    />
+
+                    <InfoRow
+                        label="Project"
+                        value={tagInfo?.projectDescription}
+                    />
+
+                    <InfoRow label="MC Pkg" value={tagInfo?.mcPkgNo} />
+
+                    <InfoRow
+                        label="Purchase order"
+                        value={tagInfo?.purchaseOrderTitle}
+                    />
+
+                    <InfoRow
+                        label="Status"
+                        value={tagInfo?.statusDescription}
+                        code={tagInfo?.statusCode}
+                    />
+
+                    <InfoRow
+                        label="Engineering"
+                        value={tagInfo?.engineeringCodeDescription}
+                        code={tagInfo?.engineeringCodeCode}
+                    />
+
+                    <InfoRow
+                        label="Contractor"
+                        value={tagInfo?.contractorDescription}
+                        code={tagInfo?.contractorCode}
+                    />
+
+                    <InfoRow
+                        label="Mounted on"
+                        value={tagInfo?.mountedOnTagNo}
+                    />
+
+                    <InfoRow label="Remark" value={tagInfo?.remark} />
+                </CollapsibleCard>
+                <CollapsibleCard cardTitle={'Details'}>
+                    {additionalFields.map((field) =>
+                        AdditionalFieldRow(
```

Comment:
Når du bruker interface i AdditionalFieldRow så må du nok endre rendring her slik som du gjorde med InfoRow. Blir komponent rendring i stedet for en funksjon.